.github/actions/version-tag/action.yml

```
@@ -0,0 +1,41 @@
+        echo "/usr/local/sbin" >> $GITHUB_PATH
+
+    - name: create new tag
+      shell: bash
+      env:
+        WORKFLOW_NAME: ${{ inputs.trigger-workflow-name }}
+      run: |
+        set +e
+        git config user.name "Bot"
+        git config user.email "admin@allthings.me"
+        currentVersion="$(git describe --tags `git rev-list --tags --max-count=1`)"
+        if [ "$currentVersion" = "" ]; then 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)